src/EnergyPlus/WindowManager.cc

```diff
@@ -6097,7 +6097,7 @@ namespace WindowManager {
 			//  ( SurfaceWindow(SurfNum)%FrameEmis*(sigma*(SurfaceWindow(SurfNum)%FrameTempSurfIn+TKelvin)**4 - rmir) + &
 			//    hcin*(SurfaceWindow(SurfNum)%FrameTempSurfIn+TKelvin - tin) )
 
-			FrameHeatGain = SurfaceWindow( SurfNum ).FrameArea * ( 1.0 + SurfaceWindow( SurfNum ).ProjCorrFrIn ) * ( hcin * ( SurfaceWindow( SurfNum ).FrameTempSurfIn + TKelvin - tin ) );
+			FrameHeatGain = SurfaceWindow( SurfNum ).FrameArea * FrameCon * ( SurfaceWindow( SurfNum ).FrameTempSurfOut - SurfaceWindow( SurfNum ).FrameTempSurfIn );
```

Comment:
I think a new report variable would be good, especially considering the current documentation says:

> ### Surface Window Heat Gain Energy [J]

> The total heat flow to the zone from the glazing, frame and divider of an exterior window when the total heat flow is positive.

> For a window without an interior shading device, this heat flow is equal to:

> - Surface Window Transmitted Solar Radiation Rate (see definition, above)
> - Convective heat flow to the zone from the zone side of the glazing
> - Net IR heat flow to the zone from zone side of the glazing
> - Short-wave radiation from zone transmitted back out the window
> - Conduction to zone from window frame and divider, if present

Currently this is not the case. This variable (and all related variables) use the [convection from the frame](https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/WindowManager.cc#L6108), and the [conduction from the divider](https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/WindowManager.cc#L6165).

I would suggest that we use the convection from both the frame and divider for these variables and create separate report variables for "Surface Window Net Heat Transfer Rate [W]", and "Surface Window Net Heat Transfer Energy [J]" using the conduction from both the frame and divider.